src/pages/Pools/Pair/index.tsx

```diff
@@ -100,67 +61,129 @@ export default function Pair({
-                      )}
-                    </Text>
-                  </Box>
-                  <Box>
-                    <ChevronDown size={12} />
-                  </Box>
-                </PointableFlex>
-              </Flex>
-              <ButtonRow>
-                <ResponsiveButtonPrimary id="join-pool-button" as={Link} padding="8px 14px" to="/create">
-                  <Text fontWeight={700} fontSize={12}>
-                    CREATE PAIR
-                  </Text>
-                </ResponsiveButtonPrimary>
-                {liquidityMiningEnabled && (
-                  <ResponsiveButtonSecondary as={Link} padding="8px 14px" to="/liquidity-mining/create">
-                    <Text fontWeight={700} fontSize={12} lineHeight="15px">
-                      CREATE REWARDS
-                    </Text>
-                  </ResponsiveButtonSecondary>
-                )}
-              </ButtonRow>
-            </TitleRow>
-            <PairView loading={wrappedPair[1] === null} pair={wrappedPair[1]} />
-          </AutoColumn>
-        </AutoColumn>
+                  <ButtonBadge to={'#'} number={0} disabled>
+                    {t('governance')}
+                  </ButtonBadge>
+                </Flex>
+              </DimBlurBgBox>
+            </TwoColumnsGrid>
+            <UserLiquidity pair={wrappedPair[1] || undefined} />
+          </ContentGrid>
+          <Flex my={3}>
+            <ButtonBadge
+              number={miningCampaigns.active.length}
+              color={miningCampaigns.active.length > 0 ? 'green' : 'orange'}
+              to={token0 && token1 ? `/rewards/${token0.address}/${token1.address}` : ''}
+              fit
+            >
+              {t('campaigns')}
+            </ButtonBadge>
+          </Flex>
+          {!loadingPairs ? <List items={miningCampaigns.active} /> : <List loading loadingItems={3} />}
+        </Box>
       </PageWrapper>
       <PairSearchModal isOpen={openPairsModal} onDismiss={handleModalClose} onPairSelect={handlePairSelect} />
     </>
   )
 }
+
+const TitleRow = styled(RowBetween)`
+  margin-bottom: 24px;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    flex-wrap: wrap;
+    gap: 12px;
+    width: 100%;
+    flex-direction: column-reverse;
+  `};
+`
+
+const PointableFlex = styled(Flex)`
+  border: solid 1px ${props => props.theme.bg3};
+  border-radius: 8px;
+  height: 36px;
+  align-items: center;
+  padding: 0 10px;
+  cursor: pointer;
+`
+
+const ButtonRow = styled(RowFixed)`
+  gap: 12px;
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    width: 100%;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-bottom: 8px;
+  `};
+`
+
+const TwoColumnsGrid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 0.4fr;
+  grid-gap: 24px;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    grid-template-columns: 1fr;
+  `};
+`
+
+const ContentGrid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 24px;
+`
```

Comment:
I think we should be consistent with rest of codebase here and move code up...

Reply:
But It's still inconsistent. Sometimes it's up and other times is at the bottom. And a few times both.

_I personally like at the bottom, because when I enter the file usually I look for the main component and if I want to check styles I'll scroll to bottom otherwise I always have to skim through a bunch of styles._

But we need to decide this as team. I think we will work on code debt in the soonish and will address this issues consistently throughout the codebase.

Reply:
yeah I also agree it is better to be at the bottom but its important to be consistent with rest of code which is in majority on the top. But if we all agree I don't mind having styles at the bottom always

Reply:
Yes. I think we should decide as a team and tackle the project whole, and make it consistent across the whole codebase in the next tasks. I'll left it as they currently are.

Reply:
I'd put these into separate file.

Reply:
I'll do that.